tests/datasource/data_connector/test_configured_asset_azure_data_connector.py

```diff
@@ -0,0 +1,1166 @@
+import os
+
+import pytest
+from azure.storage.blob import BlobServiceClient, ContainerClient
+
+from great_expectations.datasource.data_connector import (
+    ConfiguredAssetAzureDataConnector,
+)
+from great_expectations.datasource.data_connector.util import list_azure_keys
+
+try:
+    ACCOUNT_URL = os.environ["ACCOUNT_URL"]
+    CONTAINER_NAME = os.environ["CONTAINER_NAME"]
+except:
+    print("Please set environment variables before running tests")
+
+
+@pytest.fixture
+def blob_service_client() -> BlobServiceClient:
+    azure = BlobServiceClient(ACCOUNT_URL)
+    return azure
+
+
+@pytest.fixture
+def container_client(blob_service_client: BlobServiceClient) -> ContainerClient:
+    container = blob_service_client.get_container_client(CONTAINER_NAME)
+    return container
+
+
+def test_container_connects(container_client: ContainerClient):
+    blobs = [blob for blob in container_client.list_blobs()]
+    assert len(blobs) == 6
+
+
+# def test_constructor(container):
```

Comment:
Everything here and below is from S3; just pasted it here to make comparing tests easier.